### PR TITLE
make mktemp in integration tests work on OS X

### PR DESCRIPTION
**Pull request by jmhodges**

### CODE REVIEW

**Comment by jsha** [comment]
LGTM. Merge when Travis tests pass (we only have a single review requirement for client code).

File: tests/integration/_common.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 if [ "xxx$root" = "xxx" ];
 then
-    root="$(mktemp -d)"
+    root="$(mktemp -d -t leitXXXX)"
```
Comment (kuba) [inline tests/integration/_common.sh]: it's not obvious that this `-t leitXXXX` hack fixes Mac OS X. please put down a comment

Comment (jmhodges) [inline tests/integration/_common.sh]: I have done so.

**Comment by kuba** [comment]
thanks, LGTM

**Pull request merged**